cypress/integration/page.spec.js

```diff
@@ -949,23 +1027,23 @@
     });
 
     it('should allow to enter an author name', () => {
-      newCommentForm.nameInput().type('Some name')
+      newCommentForm.nameInput().type('Some name');
       newCommentForm.nameInput().should('have.value', 'Some name');
     });
 
     it('should allow to enter an author email', () => {
-      newCommentForm.emailInput().type('some@email.com')
+      newCommentForm.emailInput().type('some@email.com');
```

Comment:
There is a missing semicolon at the end of the `cy.type` statement. Ensure that each statement is properly terminated to avoid syntax errors.